diagnostic_feedback/tests/integration/base_test.py

```diff
@@ -0,0 +1,183 @@
+
+    def _get_next_button(self):
+        return self.browser.find_elements_by_xpath("//ul[@role='menu']/li")[1]
+
+    def _get_next_button_link(self):
+        return self.browser.find_element_by_xpath("//ul[@role='menu']/li/a[@href='#next']")
+
+    def wait_until_disabled(self, elem):
+        """ Wait until the given element is disabled """
+
+        wait = WebDriverWait(elem, self.timeout)
+        wait.until(lambda e: elem.get_attribute('class') == 'disabled',
+                   u"{} should be disabled".format(elem.text))
+
+    def wait_until_enabled(self, elem):
+        """ Wait until the given element is enabled """
+
+        wait = WebDriverWait(elem, self.timeout)
+        wait.until(lambda e: elem.get_attribute('class') == '',
+                   u"{} should be enabled".format(elem.text))
+
+    def load_scenario(self, xml_file, params=None, load_immediately=True):
+        """
+        Given the name of an XML file in the xml_templates folder, load it into the workbench.
+        """
+        params = params or {}
+        scenario = loader.render_template("xml_templates/{}".format(xml_file), params)
+        self.set_scenario_xml(scenario)
+        if load_immediately:
+            return self.go_to_view("student_view")
+
+    def load_student_view(self, xml_file, params=None, load_immediately=True):
+        """
+        Load student view in workbench
+        """
+        params = params or {}
+        scenario = loader.render_template("xml_templates/student_{}.xml".format(xml_file), params)
+        self.set_scenario_xml(scenario)
+        if load_immediately:
+            return self.go_to_view("student_view")
+
+    def load_studio_view(self, xml_file, params=None):
+        """
+        Load studio view in workbench
+        """
+        params = params or {}
+        scenario = loader.render_template("xml_templates/studio_{}.xml".format(xml_file), params)
+        self.set_scenario_xml(scenario)
+        return self.go_to_view("studio_view")
+
+
+class StudentBaseTest(DiagnosticFeedbackBaseTest):
+
+    def _get_startover_button(self):
+        return self.browser.find_elements_by_xpath("//ul[@role='menu']/li")[3]
+
+    def _get_startover_button_link(self):
+        return self.browser.find_element_by_xpath("//ul[@role='menu']/li/a[@href='#cancel']")
+
+    def _is_any_choice_selected(self):
+        selected = False
+        choices = [c for c in self.browser.find_elements_by_xpath("//input[@type='radio']") if c.is_displayed()]
+
+        for choice in choices:
+            if choice.is_selected():
+                selected = True
+                break
+
+        return selected
+
+    def _get_choice(self, idx):
+        return [c for c in self.browser.find_elements_by_xpath("//input[@type='radio']") if c.is_displayed()][idx]
+
+    def _verify_question(self, question_order, next_btn, back_btn, choice_idx):
+
+        self.assertEqual(self._is_any_choice_selected(), False)
```

Comment:
@jareerahsan You should use `self.assertFalse` for checking if a value is `False`; it's a bit shorter and more to the point. It would be worth checking the rest of the tests to see if there are other instances of this and update them as well.